Midtrans/Transaction.php

```diff
@@ -81,7 +81,7 @@ public static function expire($id)
      * 
      * @return mixed[]
      */
-    public static function refund($id)
+    public static function refund($id, $params)
     {
         return ApiRequestor::post(
```

Comment:
This function looks like having bugs, because it does not send the $params to `ApiRequestor`